west.yml

```diff
@@ -100,7 +100,7 @@ manifest:
       revision: f49bd1354616fae4093bf36e5eaee43c51a55127
       path: tools/net-tools
     - name: hal_nxp
-      revision: 0d11138724959e1162777d9206f841ccdf64348e
+      revision: pull/90/head
```

Comment:
Any progress on getting this merged?

Reply:
I have pinged the team for their review comments, will update this once the PR is merged.

Reply:
Done

Reply:
This has been updated.